### ISSUE TITLE
Negative article count

### ISSUE THREAD

**Issue by PrestonMcAfee** [sismics/reader#164]
Since V1.5, I've started to see negative article counts.  This happens when I refresh and some new, unread articles appear in Unread but are not reflected in the article count, that is, I have more unread articles than the app reflects.  Once read, these show up as negative numbers of articles, see attached image.
<img width="260" alt="capture" src="https://user-images.githubusercontent.com/41998813/43522493-657c76d4-954e-11e8-8e48-d14354c2daf1.PNG">

I love sismics!

**Comment by jendib** [comment]
Yes we are aware of this bug, it's related to the feature updating the article if the feed is updated. It's just a counting error and no article will be missing. We will fix it soon hopefully, thanks for the report!

**Comment by PrestonMcAfee** [comment]
Thanks!  I realized I didn't lose any articles (I compared against
Aggregator, which I use as a backup), and so no rush to fix.  I *love*
sismics and even posted a blurb on it on my website:

https://mcafee.cc/Bin/Sismics.html

Preston McAfee

**Comment by jendib** [comment]
Great article, thanks a lot :) We are not adding a bunch of features (lack of time...) but we still use this everyday as our main source of news, so no worries for the future support 👍

**Comment by jendib** [comment]
Are you the sole user of your Reader server, or is it shared with someone else? If yes, does this person is also subscribed to Gizmodo?

**Comment by naku** [comment]
Related to #128 ?
Might happen when an article is edited in the RSS stream
